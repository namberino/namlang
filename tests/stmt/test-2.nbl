var a = "hello";
print(a); // hello
var a = "world";
print(a); // world
var c;
print(c); // "nil".
