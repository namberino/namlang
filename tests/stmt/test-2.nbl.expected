Executing file: tests/stmt/test-2.nbl
hello
world
nil
